Dockerfile

```diff
@@ -20,7 +20,6 @@ ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
 
 RUN apt-get update && \
     apt-get -y upgrade && \
-    apt-get install -y curl && \
```

Comment:
should this come from main branch?

Reply:
Based on discussion with Aled, this is to be removed as per his chat with Peter Philips.